esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -458,6 +458,15 @@ void CWsWorkunitsEx::init(IPropertyTree *cfg, const char *process, const char *s
 
     const char *name = cfg->queryProp("Software/EspProcess/@name");
     getConfigurationDirectory(directories, "query", "esp", name ? name : "esp", queryDirectory);
+    if (queryDirectory.isEmpty() && isContainerized())
+    {
+        const char *dllserver_root = getenv("HPCC_DLLSERVER_PATH");
```

Comment:
This environment variable isn't defined in the containerized version any more.  It should always be provided via a storage plane.  Do you know why this code was added?